.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Stale issue handler'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 7 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.0.0
+        id: stale
+        with:
+          stale-issue-message: 'This Issue will be closed in 30 days. Please remove the "Stale" label or comment to avoid closure with no action.'
+          stale-pr-message: 'This PR will be closed in 30 days. Please remove the "Stale" label or comment to avoid closure with no action.'
+          days-before-stale: 300
+          days-before-close: 30
+          stale-issue-label: 'stale'
+          stale-pr-label: 'stale'
+          close-issue-label: 'closed'
```

Comment:
I don't think closed issues/prs need a label.

Reply:
I added this so that we can track (at least initially) the set of issues / PR's that get closed automatically vs all of the others that we have and will close manually (which won't have these labels). According to the documentation the action will also remove this if it gets re-opened.

So eventually (once we are in a much better sate) I agree this will not be needed but as we currently have 117 open (most of which I expect will eventually get closed by this) and 622 Closed I want an easier way to identify these auto-closed items.